appengine/standard/xmpp_wikibot/wikibot.py

```diff
@@ -0,0 +1,207 @@
+# Copyright 2016 Google Inc. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Wikibot server example using SleekXMPP client library"""
+
+import sys
```

Comment:
Imports need to be two sections: standard lib, then thirdparty and should be in alphabetical order:

```
import getpass
import json
import logging
import optparse
import sys
import threading
import urllib

from flask import Flask, request
import requests
import sleekxmpp
```